packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -1033,10 +1033,12 @@ export class ArcRotateCamera extends TargetCamera {
             this.radius += (this._goalRadius - this.radius) * t;
             Vector2.LerpToRef(this.targetScreenOffset, this._goalTargetScreenOffset, t, this.targetScreenOffset);
 
+            TmpVectors.Quaternion[0].negateToRef(TmpVectors.Quaternion[1]);
+
             // stop restoring when within close range or when user starts interacting
             if (
                 (Vector3.DistanceSquared(this.getTarget(), this._goalTarget) < Epsilon &&
-                    TmpVectors.Quaternion[2].equalsWithEpsilon(TmpVectors.Quaternion[0]) &&
+                    (TmpVectors.Quaternion[2].equalsWithEpsilon(TmpVectors.Quaternion[0]) || TmpVectors.Quaternion[2].equalsWithEpsilon(TmpVectors.Quaternion[1])) &&

```

Comment:
I wonder if Quaternion.equals* should do this, possibly behind a flag?

Reply:
I've been thinking about that and I'd be in favor of adding a new method instead: no flag check, more explicit naming and dedicated comment.

Reply:
I've added a new method instead.